website/thaliawebsite/management/commands/migratemedia.py

```diff
@@ -24,13 +24,21 @@ def handle(self, *args, **options):
 
         # process migrate local media files to s3
         for full_path in self._get_all_media_file():
+            upload_path = self._split_path_to_upload(full_path)
             try:
-                upload_path = self._split_path_to_upload(full_path)
-                s3_client.upload_file(
-                    full_path, settings.AWS_STORAGE_BUCKET_NAME, upload_path
-                )
-                print(f"success upload {upload_path}")
-                logging.info(f"success upload {upload_path}")
+                if s3_client.head_object(
```

Comment:
From https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/head_object.html I gather head_object will raise if the key doesn't exist. So this return value probably shouldn't be used as boolean